diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3a84d9bb3..554d45bfe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -552,8 +552,7 @@ if(CONFIG_AV1_ENCODER)
       if(BUILD_SHARED_LIBS)
         target_link_libraries(aom PRIVATE ${VMAF_LDFLAGS} ${VMAF_LIBRARIES})
       else()
-        target_link_libraries(aom
-                              PRIVATE ${VMAF_LDFLAGS} ${VMAF_LIBRARIES} -static)
+        target_link_libraries(aom PRIVATE ${VMAF_LDFLAGS} ${VMAF_LIBRARIES})
       endif()
       target_include_directories(aom PRIVATE ${VMAF_INCLUDE_DIRS})
       if(VMAF_CFLAGS)
diff --git a/aom_ports/aom_ports.cmake b/aom_ports/aom_ports.cmake
index e02157a6c..3ac64714b 100644
--- a/aom_ports/aom_ports.cmake
+++ b/aom_ports/aom_ports.cmake
@@ -77,9 +77,9 @@ function(setup_aom_ports_targets)
 
     if("${AOM_TARGET_CPU}" STREQUAL "x86"
        OR "${AOM_TARGET_CPU}" STREQUAL "x86_64")
-      target_sources(aom PRIVATE ${AOM_PORTS_INCLUDES_X86})
+      #target_sources(aom PRIVATE ${AOM_PORTS_INCLUDES_X86})
       if(BUILD_SHARED_LIBS)
-        target_sources(aom_static PRIVATE ${AOM_PORTS_INCLUDES_X86})
+        #target_sources(aom_static PRIVATE ${AOM_PORTS_INCLUDES_X86})
       endif()
     endif()
   endif()
